src/main/java/frc/robot/config/Config.java

```diff
@@ -37,12 +37,38 @@
     /**
      * PLACE IDS OF ROBOTS HERE
      **/
-
     // Mergio is has the ID 2
+
+    // This is a static class which should not be instantiated
     private Config() {
 
     }
 
+    // Static Constants
+    public static int RIGHT_FRONT_TALON = robotSpecific(3, 3, 3);
+    public static int RIGHT_REAR_TALON = robotSpecific(4, 4, 4);
+    public static int LEFT_FRONT_TALON = robotSpecific(1, 1, 1);
+    public static int LEFT_REAR_TALON = robotSpecific(2, 2, 2);
+
+    public static Double DRIVE_OPEN_LOOP_DEADBAND = 0.04;
+
+    public static Double JOYSTICK_AXIS_DEADBAND = 0.1;
+
+    public static int LEFT_CONTROL_STICK_Y = 1;
+    public static int LEFT_CONTROL_STICK_X = 0;
+
+    public static int RIGHT_CONTROL_STICK_Y = 5;
+    public static int RIGHT_CONTROL_STICK_X = 4;
+
+    public static boolean INVERT_FORWARD = robotSpecific(true, true, true);
+    public static boolean INVERT_SIDE= robotSpecific(false, false, false);
+
+
+    // Timeouts for sending CAN bus commands
+    public static final int
+            CAN_SHORT = 10,
```

Comment:
Would be a little clearer to use the same style and have explicit declarations on one line. Also having TIMEOUT in the name would clarify this when it is used in other places in the codebase